packages/flutter_tools/pubspec.yaml

```diff
@@ -18,7 +18,7 @@ dependencies:
   intl: 0.15.2
   json_rpc_2: 2.0.4
   json_schema: 1.0.6
-  linter: 0.1.40
+  linter: 0.1.41
```

Comment:
?

Reply:
This should surely have affected more than just this package.

Reply:
I ran update-packages, and linter was just updated between the time I submitted the previous update-packages CL and this one